enhancements/microshift/microshift-updateability-ostree.md

```diff
@@ -417,8 +410,7 @@ from clean state.
 ##### Backup management
 
 - "Data" refers to MicroShift's data
-- Terms such as _previous boot_, _previous boot's commit_, and
-  _previous commit with MicroShift_ are related to MicroShift's "history of commits"
+- _previous commit_ are related to MicroShift's health metadata
```

Comment:
Do we want to use the `deployment` terminology here as well?

Reply:
Yes, I changed other "commits" where relevant